org.lflang/src/org/lflang/generator/python/PythonGenerator.xtend

```diff
@@ -1911,10 +1911,9 @@ class PythonGenerator extends CGenerator {
             # For instructions, see: https://github.com/icyphy/lingua-franca/wiki/Containerized-Execution
             FROM python:alpine
             WORKDIR /lingua-franca/«topLevelName»
-            COPY . src-gen
             RUN set -ex && apk add --no-cache gcc musl-dev \
-             && cd src-gen && python3 setup.py install && cd .. \
-             && apk del gcc musl-dev
```

Comment:
I think the purpose of deleting things like `gcc` is to minimize the final image size. Are these dependencies still needed after the `python3 setup.py install`?

Reply:
There is a tradeoff between build time and image size. Docker treats each RUN instruction as a "layer" and caches them. If I delete the dependencies, every build of a new lf program needs to re-add those dependencies whereas if I don't delete them, docker will use the cached layer.

Reply:
Interesting. Perhaps speeding up subsequent build times makes more sense for the Python target then.